Dockerfile.vanilla

```diff
@@ -0,0 +1,30 @@
+FROM paritytech/ci-linux:production as builder
```

Comment:
Can we merge these two Dockerfiles into one?

Reply:
the default command isn't the same